src/docgen/render/markdown-render.ts

```diff
@@ -630,7 +630,9 @@ export class MarkdownRenderer {
     }
 
     for (const [key, value] of Object.entries(metadata)) {
-      md.bullet(`${MarkdownDocument.italic(`${key}:`)} ${value}`);
+      // To denote if a parameter is variadic
+      const variadicMarker = parameter.variadic ? 'variadic ' : '';
+      md.bullet(`${MarkdownDocument.italic(`${key}:`)} ${variadicMarker}${value}`);
```

Comment:
Not sure if this is exactly how we want to do this - right now I set this so docs show a parameter is variadic like so:

**Type:** variadic string

I could change it to be each language's specific way for declaring a variadic parameter

Reply:
Yes this should be language specific. Might be a little tricky. lets talk.